golang/1.9/go-ast-scanner

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p /log
+gas -exclude=$GLOBAL_WHITELIST -fmt=junit-xml ./... | go run /gas-filter-report.go -whitelist whitelist.json > /log/report.xml
```

Comment:
What do you think about putting this into new repo, `go get -u <new repo>` which will compile the binary, then running the binary in the scanner?

Reply:
ahh sounds like a good idea.